Libraries/MiscDrivers/Camera/ov5640.c

```diff
@@ -511,30 +511,26 @@ static int reset(void)
             MXC_Delay(MSEC(default_regs[i].val));
         } else {
             ret |= cambus_write(default_regs[i].addr, (uint8_t)default_regs[i].val);
+#if 0
```

Comment:
Consider making a definition to enable and disable this section.  (#ifdef VERIFY_REGISTER_SETTINGS ?)

Reply:
https://github.com/Analog-Devices-MSDK/msdk/pull/583/commits/121d0f204ad2c7c7d711b265c2e17f0c80b7babc